jhotdraw-utils/src/main/java/org/jhotdraw/geom/ConvexHull.java

```diff
@@ -98,26 +98,6 @@ public static Path2D.Double getConvexHullPath2D(Shape shape) {
     return convexHull;
   }
 
-  /**
-   * Computes the convex hull from a set of points.
-   *
-   * @param points
-   * @return convex hull of the points
-   */
-  public static List<Point> getConvexHull(List<Point> points) {
-    return Arrays.asList(getConvexHull(points.toArray(new Point[points.size()])));
-  }
-
-  /**
-   * Computes the convex hull from a set of points.
-   *
-   * @param points
-   * @return convex hull of the points
-   */
-  public static List<Point2D.Double> getConvexHull2D(List<Point2D.Double> points) {
-    return Arrays.asList(getConvexHull2D(points.toArray(new Point2D.Double[points.size()])));
-  }
-
```

Comment:
It is a good practice to remove the unused methods. It keeps the code clean and avoid unnecessary confusion. 
We can merge this code.